res/openvpn-gui-res-ua.rc

```diff
@@ -0,0 +1,341 @@
+/*
+ *  OpenVPN-GUI -- A Windows GUI for OpenVPN.
+ *
+ *  Copyright (C) 2004 Mathias Sundman <mathias@nilings.se>
+ *                2015 http://picku.pp.ua
```

Comment:
This copyright entry is fine, because this is the translation file. I reword it like this:

Translation copyright (C) 2015 http://picku.pp.ua